pset-iterators.js

```diff
@@ -42,7 +42,11 @@ Expected Result Shape: A string representing the full name of the president that
 console.log('Problem 1:')
 
 // Add your code below this line
+const firstWhigPresident = presidents.find(function (firstWhigMember) {
+  return firstWhigMember.party === "Whig"
+})
 
+console.log(firstWhigPresident.president)
```

Comment:
@OcielC Great work with this solution to use the `find` method to return the first president of the `Whig` party. 

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L35)